doc/Network.mdwn

```diff
@@ -1,29 +1,57 @@
 # Network infrastructure
 
-## VLANs
+## Networks
 
-NOC runs 2 core switches (one in each room), carrying a bunch of VLANs:
-- 33 is the management VLAN (192.168.33.0/24);
-- 127 is the LAN (192.168.127.0/24);
-- 255 (`0xFF`) is our Funkfeuer VLAN.
+NOC operates a number of networks, available as tagged VLANs on the core
+switches (one in each half of the hackerspace).  These networks are:
 
-The switches have hostnames `sw{0,1}.mgmt.realraum.at`, and the WiFi access
-points are similarly `ap{0,1}.mgmt.realraum.at`. `0` denotes the main room, and
-`1` denotes Wöhnung 2.
+| Network    | VLAN id | Extra subnets    |
+|------------|---------|------------------|
+| Management |      23 | --               |
+| IoT        |      33 | --               |
+| Services   |      34 | --               |
```

Comment:
Net Range 192.168.34.0/24 is currently used for connections on http bridge on Alfred... Not sure if we want to make this the service VLAN....

Reply:
IIRC, that was the proposal when we discussed (and what I wrote on paper).

To clarify (I just realised this needs some extra explanatory text):
- services reachable from outside (so, Nginx & friends) live in the “public” VLAN;
- the “services” VLAN is for not-frontend services, that aren't reachable from anywhere else (unless they explicitly have an extra interface).

Reply:
Would naming it “backend” or somesuch be clearer?